model-mesh-mlserver-adapter/server/adaptmodellayout_test.go

```diff
@@ -405,6 +405,36 @@ var adaptModelLayoutTests = []adaptModelLayoutTestCase{
 			},
 		},
 	},
+
+	// model with filename that alphabetically precedes model-settings.json
+
+	{
+		ModelID:   "model-filename-precedes-model-settings",
+		ModelType: "sklearn",
+		ModelPath: "data",
+		InputFiles: []string{
+			"data/model-settings.json",
+			"data/aaaaa.json",
```

Comment:
I generally would agree, but here I think the `aaaaa` drives home the point of this being a alphabetical sorting related issue

Reply:
Fair enough! I went back and forth on it myself and figured [test case's description above](https://github.com/kserve/modelmesh-runtime-adapter/pull/43/files#diff-7bc70639f032bdbdc7d6e0769fd075a00ab7b634bb7ef2db88331c613d517beeR409) it could be enough. Your call :)